modules/service/src/main/scala/lucuma/odb/graphql/input/ObsAttachmentPropertiesInput.scala

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) 2016-2023 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package lucuma.odb.graphql
+
+package input
+
+import cats.syntax.all._
+import eu.timepit.refined.types.string.NonEmptyString
+import lucuma.odb.data.Nullable
+import lucuma.odb.data.Tag
+import lucuma.odb.graphql.binding._
+
+object ObsAttachmentPropertiesInput {
+
+  case class Edit(
+    description: Nullable[NonEmptyString],
+    checked: Option[Boolean]
+  )
```

Comment:
There's only `Edit` because creation is handled by the REST API, correct?

Reply:
Correct. The REST API is the only place that knows how to deal with S3. So creation and delete only happen there.